regression/ansi-c/array_initialization4/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+main.c
+
+^EXIT=(1|64)$
```

Comment:
:question: Why ambiguity here?

Reply:
That's consistency with Visual Studio and GCC, which each have different exit codes.